src/Data/Fin/Base.agda

```diff
@@ -335,3 +332,18 @@ NB argument order has been flipped:
 the left-hand argument is the Fin m
 the right-hand is the Nat index increment."
 #-}
+------------------------------------------------------------------------
+-- _≺_
```

Comment:
We tend to only use section headings in the deprecation sections to denote versions. Could you remove this one?